opensciencegrid/ospool-cm/ospool.cron

```diff
@@ -0,0 +1,8 @@
+
+# prios
+ */5 * * * *  root  /opt/ospool/update-prios >/dev/null 2>&1
+
+# web
+ */5 * * * *  root  cd /shared/web/overview && ./utils/cron-5m.sh >/dev/null 2>&1
+  0  * * * *  root  cd /shared/web/overview && ./utils/cron-hourly.sh >/dev/null 2>&1
```

Comment:
Seems like an unrelated change that should at the very least go into a separate commit

Reply:
Removed in new commit.